src/test/java/teammates/test/cases/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -227,8 +230,12 @@ private void testSubmitAction() throws Exception {
         assertNull(BackDoor.getFeedbackResponse(fqContrib.getId(),
                                                 "SFSubmitUiT.alice.b@gmail.tmt",
                                                 "SFSubmitUiT.benny.c@gmail.tmt"));
+
         submitPage.submitWithoutConfirmationEmail();
 
+        confirmationEmailBox = submitPage.isConfirmationEmailBoxTicked();
+        assertFalse(confirmationEmailBox);
```

Comment:
These two lines can be combined to `assertFalse(submitPage.isConfirmationEmailBoxTicked());`